### DISCUSSION TITLE
Set views for user group (automatically)

### DISCUSSION THREAD

**Discussion by kouata** [gocd/gocd#11019] [Q&A]
Hi everyone,
I'd like each user to be able to see only the pipelines relevant to them. Does anyone know how to delete the default view,  create a nvew view and associate it with a user group ? Is it possible to automate this (maybe editing /etc/go/cruise-config.xml ?) ?
once the user is logged in, he should see the view that concerns him and not the default view.
Thank you !

**Comment by chadlwilson** [comment]
I dont believe this is possible via views. 

You can use group permissions to remove the view permission for groups of pipelines that are not relevant to a user however.
